app/src/main/java/eu/kanade/tachiyomi/ui/base/FabController.kt

```diff
@@ -0,0 +1,10 @@
+package eu.kanade.tachiyomi.ui.base.controller
+
+import com.google.android.material.floatingactionbutton.ExtendedFloatingActionButton
+
+interface FabController {
+
+    fun configureFab(fab: ExtendedFloatingActionButton) {}
+
+    fun cleanupFab(fab: ExtendedFloatingActionButton) {}
+}
```

Comment:
so the reason this was added was because I was too monke to figure out line 28 and remove it
https://github.com/Jays2Kings/tachiyomiJ2K/blob/7307da5050d584f47f80bc0706704b612b9afa2a/app/src/main/java/eu/kanade/tachiyomi/ui/setting/database/ClearDatabaseController.kt#L26-L32